### PR TITLE
[script.speedfaninfo] 1.1.100

### PR DESCRIPTION
**Pull request by pkscout**
### Description
Matrix specific version. No changes to codereviewed in PR #1309, only updated python version requirement and bumped version number.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.speedfaninfo/addon.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		<description lang="sk_SK">Rozoberá a zobrazuje informácie zo SpeedFan logu. Síce tento doplnok funguje na všetkých platformách, SpeedFan funguje len na Windows, a preto je tento doplnok určený užívateľom Kodi na Windows. Prosím prečítajte readme.txt aby ste sa dozvedeli ako nastaviť SpeedFan log tak, aby ho tento doplnok vedel správne rozoberať.</description>
+		<description lang="sv_SE">Analyserar och visar information från SpeedFan loggen. Trots att detta tillägg fungerar på alla plattformar, fungerar SpeedFan endast på Windows, så det är optimerat för folk som använder Kodi på Windows. Läs readme.txt för mer information hur du konfigurerar din SpeedFan logg så att tillägget kan analysera det riktigt.</description>
+		<description lang="zh_CN">分析和显示从SpeedFan日志获取的信息。由于SpeedFan只有Windows版，因此本插件仅用于运行在Windows平台的Kodi。请阅读readme.txt中关于如何配置SpeedFan日志以使本插件能正确分析处理的相关说明。</description>
+		<platform>all</platform>
```
[the remark this answers is not in the document]
Comment (pkscout) [reply script.speedfaninfo/addon.xml]: The add-on runs on any platform (it only reads the log file from the Windows program). So, for instance, I run the add-on on my Vero 4K+ (running OSMC) and connect to a Windows share so I can read the log file.

File: script.speedfaninfo/addon.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+		<license>GPL-2.0-only</license>
+		<forum>https://forum.kodi.tv/showthread.php?tid=122481</forum>
+		<website>https://kodi.wiki/view/Add-on:SpeedFan_Information_Display</website>
+		<email>speedfaninfo.xbmc@gmail.com</email>
+		<source>https://github.com/pkscout/script.speedfaninfo</source>
+	</extension>
```
[the remark this answers is not in the document]
Comment (pkscout) [reply script.speedfaninfo/addon.xml]: It's already there (at least from what I can tell) on lines 17-19 (right after the news section).

File: script.speedfaninfo/addon.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (romanvm) [reply script.speedfaninfo/addon.xml]: ok then.

File: script.speedfaninfo/addon.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (romanvm) [reply script.speedfaninfo/addon.xml]: Ok, sorry for missing that.

**Comment by romanvm** [comment]
LGTM

**Pull request merged**